pkg/cmd/service/service.go

```diff
@@ -45,13 +46,13 @@ type Config struct {
 	// RequestTimeout is a limit on the time a request may take. If the value is 0, then there is no timeout.
 	RequestTimeout time.Duration `default:"0s" split_words:"true"`
 
-	GRPCTLSEnabled  bool   `default:"false" envconfig:"GRPC_TLS_ENABLED"`
-	GRPCTLSCertPath string `envconfig:"GRPC_TLS_CERT_PATH"`
-	GRPCTLSKeyPath  string `envconfig:"GRPC_TLS_KEY_PATH"`
+	GRPCTLSEnabled  bool   `default:"false" envconfig:"OPENFGA_GRPC_TLS_ENABLED"`
```

Comment:
Does this play well with the `prefix` in which it's read?

Also why not continue to use `split_words` instead of mixing 🤔

Reply:
I'm not sure at what point we started using the `envconfig` struct tag in favor of the `split_words`, but I was talking with @craigpastro about this yesterday. I guess the issue with `split_words` is it is sometimes ambiguous where to split on, and `envconfig.Process` doesn't always split it the way we want it, so we've chosen to be explicit for these fields. The splitting typically works on change in upper/lower character case, but since we have the `stylecheck` lint checks our field names end up have less change in character case. 

Another thing I noticed is that if you use the `envconfig` struct tag to specify what env variable to use for the field, then `default` value struct tags get applied to the env variable *before* the prefix is applied. Default values don't work well if you're using a prefix + explicit env variable name. Adding the expected prefix in front of the explicit env variable name fixes the issue.